scripts/postprocess_sitemap.py

```diff
@@ -0,0 +1,25 @@
+from bs4 import BeautifulSoup
+
+with open('./_build/html/sitemap.xml', 'r') as f:
+    contents = f.read()
+
+soup = BeautifulSoup(contents, 'lxml')
+
+urls = soup.find_all('url')
+
+for url in urls:
+    loc = url.find('loc')
+    text = loc.string
+    # Remove the 'gen' and '404' pages
+    if '404' in text:
+        url.decompose()
+        continue
+    if 'genindex.html' in text:
+        loc.string = text[:-5]  # removes the ".html"
+    elif text.endswith('index.html'):
+        loc.string = text[:-10]  # removes the "index.html"
+    elif text.endswith('.html'):
+        loc.string = text[:-5]  # removes the ".html"
+
+with open('./_build/html/sitemap.xml', 'w') as f:
+    f.write(str(soup))
```

Comment:
The unhappy looking "stop" symbol at the end of the file probably means it's lacking a newline character on the last line. It would be better to add one.

Reply:
I add `f.write('\n')` or just extra line in 26 but still seeing this "stop" symbol. Not an issue locally in my visual studio code. Any idea?

Reply:
Works now :) was not earlier for some reason ;)